docs/manuel/src/01-frontend/style.md

```diff
@@ -0,0 +1,3 @@
+# Style
+
+Le thème de la langate est réalisé en CSS.
```

Comment:
One line, one error :wink: 
The front is made with tailwind without css classes.

Reply:
umm... it's placeholder anyway 😃👍